packages/notifi-react-card/lib/components/subscription/EventTypeBroadcastRow.tsx

```diff
@@ -46,30 +54,71 @@ export const EventTypeBroadcastRow: React.FC<EventTypeBroadcastRowProps> = ({
   }, [alertName, config, inputs]);
   const tooltipContent = config.tooltipContent;
 
+  const didFetch = useRef(false);
+
   useEffect(() => {
-    if (loading) {
+    if (didFetch.current) {
       return;
     }
+
     const hasAlert = alerts[alertName] !== undefined;
     setEnabled(hasAlert);
+    didFetch.current = true;
   }, [alertName, alerts, loading]);
```

Comment:
we can remove `loading` from dependencies i think

(and do we need to add the ref? technically i don't think we do)

Reply:
i think i had to ref to prevent really quick flickering on the toggles. let me remove and check again

Reply:
yah, w/o the ref, it flickers